docs/Data correction documentary-cleared.ipynb

```diff
@@ -0,0 +1,301 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "788a4fce-63dc-4c23-8d2a-e9a05db0fdf7",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "## Data correction procedure for TOF-SANS with polarization analysis\n",
+    "\n",
+    "- Note on notation:\n",
+    "> - subscripts = discretization variable from measurement\n",
+    "> - superscripts = Names\n",
+    "> - () = continuous function of....\n",
+    "\n",
+    "| Value | Meaning | shape | How acquired? | How named in current main branch? |\n",
```

Comment:
What is the last column? Can we remove it?

Reply:
That was created when I had in mind we previously named C = P_He0, and T_E = T_g, for you to know what it is. But I figured out this is the only cases. So probably not needed anymore